docs/UserGuide.md

```diff
@@ -70,30 +70,50 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a client: `add`
+### Adding a client : `add`
 
 Adds a client to the database.
 
-Format: `add n/NAME i/NRIC p/CONTACT NUMBER l/LICENCE PLATE…​`
+Format: `add n/NAME i/NRIC p/CONTACT NUMBER e/EMAIL a/ADDRESS t/TAG l/LICENCE PLATE [c/COMPANY
+ pn/POLICY NUMBER pi/POLICY ISSUE DATE pe/POLICY EXPIRY DATE]`
 
 * Add a client's details in the database.
-* **All** fields must be present when using this command.
+* **All** client details must be present when using this command.
+* If adding a client’s policy, **all** policy details must be present when using this command.
 
 Examples:
-* `add n/Mary i/627A p/91234567 l/SLU5237J` creates a client whose name is `Mary`, NRIC last four digits is `627A`, contact number is `91234567`, licence plate is `SLU5237J` in the database.
+* `add n/Irfan Ibrahim i/752X p/92492021 e/irfan@example.com a/Blk 47 Tampines Street 20,#17-35 t/classmates l/SBP8888T` adds a client Irfan without a policy.
+* `add n/Bob Anderson i/578A p/54783402 e/bobanderson@gmail.com a/Blk 233 Serangoon Avenue 3, #05-12 t/teacher l/SGP1208J c/DEF Insurance pn/263J pi/20-09-2023 pe/19-09-2024` adds a client Bob with the policy he buys.
```

Comment:
Good examples to show both cases: A client has a policy and a client who does not